lib/routes/account.js

```diff
@@ -307,6 +307,10 @@ module.exports = (log, db, mailer, Password, config, customs, signinUtils, push)
                     tokenVerificationId: tokenVerificationId
                   })
                 }
+
+                // show an error to the user, the account is already created.
+                // the user can come back later and try again.
+                throw error.cannotSendEmail()
```

Comment:
Should we also delete the account at this point? Otherwise we're just asking for unverified accounts to sit around stinking up the database, which is both a waste of space and presumably violates GDPR.

Reply:
> asking for unverified accounts to sit around stinking up the database

 don't we already have that issue for those who sign up, successfully get the email, but never verify?

Reply:
We do, but in this case at least we know we can do something about it is all I was thinking. No worries if you think it's the wrong thing to do though.